README.md

```diff
@@ -316,6 +361,27 @@ environment variable on `1` before executing the test script:
 export DEBUG=1
 ```
 
+<a name="vagrant-spec-based-tests"></a>
+#### Vagrant Spec based tests
+
+The source contains also a set of [vagrant-spec](https://github.com/mitchellh/vagrant-spec) acceptance tests. They can be run via:
+
+```
+vagrant-spec test --components provider/virtualbox/cdk
+vagrant-spec test --components provider/libvirt/cdk
+```
+
+In order to successfully run the tests, you need to set the following
+environment variables:
+
+* _VAGRANT_SPEC_BOX_VIRTUAL_BOX_ - URL to the VirtualBox box for testing
```

Comment:
@hferentschik This means an user has to download the CDK Vgrantbox from customer portal and then host it in some http location. Do we have anyway to specify if the Vagrantbox is already present on the test machine? May be a file path or config.vm.box name if the box is already installed on the  system.